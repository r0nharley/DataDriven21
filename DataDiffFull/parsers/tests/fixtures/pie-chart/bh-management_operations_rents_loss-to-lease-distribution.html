<svg version="1.1" class="highcharts-root" style="font-family:&quot;Lucida Grande&quot;, &quot;Lucida Sans Unicode&quot;, Arial, Helvetica, sans-serif;font-size:12px;" xmlns="http://www.w3.org/2000/svg" width="466" height="431">
    <desc>Created with Highstock 5.0.6</desc>
    <defs>
        <clipPath id="highcharts-k7uut2n-26">
            <rect x="0" y="0" width="446" height="406" fill="none"></rect>
        </clipPath>
    </defs>
    <rect fill="#ffffff" class="highcharts-background" x="0" y="0" width="466" height="431" rx="0" ry="0"></rect>
    <rect fill="none" class="highcharts-plot-background" x="10" y="10" width="446" height="406"></rect>
    <rect fill="none" class="highcharts-plot-border" x="10" y="10" width="446" height="406"></rect>
    <g class="highcharts-series-group">
        <g class="highcharts-series highcharts-series-0 highcharts-pie-series highcharts-color-undefined highcharts-tracker " transform="translate(10,10) scale(1 1)">
            <path fill="#00cee6" d="M 415.42813413317515 217.84631920796335 A 193 193 0 0 1 179.7140499815976 391.083296788961 L 179.7140499815976 391.083296788961 A 193 193 0 0 0 415.42813413317515 217.84631920796335 Z" class="highcharts-halo highcharts-color-2" fill-opacity="0.25"></path>
            <path fill="#6eda55" d="M 222.96069107168682 10.000004003087724 A 193 193 0 0 1 414.138523529841 176.2593039949889 L 280.3415570589523 194.97779119849667 A 57.9 57.9 0 0 0 222.98820732150605 145.10000120092633 Z" class="highcharts-point highcharts-color-0 " stroke="#ffffff" stroke-width="1" stroke-linejoin="round"></path>
            <path fill="#000000" d="M 414.16516865213543 176.4504558570092 A 193 193 0 0 1 415.4428842358497 217.65388368274253 L 280.7328652707549 207.39616510482276 A 57.9 57.9 0 0 0 280.3495505956406 195.03513675710275 Z" class="highcharts-point highcharts-color-1" stroke="#ffffff" stroke-width="1" stroke-linejoin="round"></path>
            <path fill="#00cee6" d="M 415.42813413317515 217.84631920796335 A 193 193 0 0 1 179.7140499815976 391.083296788961 L 210.0142149944793 259.4249890366883 A 57.9 57.9 0 0 0 280.72844023995253 207.453895762389 Z" class="highcharts-point highcharts-color-2 " stroke="#ffffff" stroke-width="1" stroke-linejoin="round"></path>
            <path fill="#9b9bd7" d="M 179.5259883591291 391.03991680451634 A 193 193 0 0 1 30.122236864160328 209.86793183679916 L 165.1366710592481 205.06037955103974 A 57.9 57.9 0 0 0 209.95779650773872 259.4119750413549 Z" class="highcharts-point highcharts-color-3" stroke="#ffffff" stroke-width="1" stroke-linejoin="round"></path>
            <path fill="#218A8C" d="M 30.115465372341703 209.675050671844 A 193 193 0 0 1 74.69241293239867 79.49348349910501 L 178.5077238797196 165.9480450497315 A 57.9 57.9 0 0 0 165.1346396117025 205.0025152015532 Z" class="highcharts-point highcharts-color-4" stroke="#ffffff" stroke-width="1" stroke-linejoin="round"></path>
            <path fill="#fbb755" d="M 74.81599358210246 79.34523769000847 A 193 193 0 0 1 132.78471584145603 32.382877458933876 L 195.93541475243683 151.81486323768016 A 57.9 57.9 0 0 0 178.54479807463073 165.90357130700255 Z" class="highcharts-point highcharts-color-5" stroke="#ffffff" stroke-width="1" stroke-linejoin="round"></path>
            <path fill="#fc7570" d="M 132.9553780431993 32.29274749836537 A 193 193 0 0 1 222.73192687195217 10.000186174188173 L 222.91957806158567 145.10005585225645 A 57.9 57.9 0 0 0 195.9866134129598 151.7878242495096 Z" class="highcharts-point highcharts-color-6" stroke="#ffffff" stroke-width="1" stroke-linejoin="round"></path>
        </g>
        <g class="highcharts-markers highcharts-series-0 highcharts-pie-series highcharts-color-undefined " transform="translate(10,10) scale(1 1)"></g>
    </g>
    <g class="highcharts-data-labels highcharts-series-0 highcharts-pie-series highcharts-color-undefined highcharts-tracker" transform="translate(10,10) scale(1 1)" opacity="1">
        <g class="highcharts-label highcharts-data-label highcharts-data-label-color-0 highcharts-tracker" style="" transform="translate(299,74)"></g>
        <g class="highcharts-label highcharts-data-label highcharts-data-label-color-2 highcharts-tracker" style="" transform="translate(289,320)"></g>
        <g class="highcharts-label highcharts-data-label highcharts-data-label-color-3 highcharts-tracker" style="" transform="translate(74,293)"></g>
        <g class="highcharts-label highcharts-data-label highcharts-data-label-color-4 highcharts-tracker" style="" transform="translate(53,142)"></g>
        <g class="highcharts-label highcharts-data-label highcharts-data-label-color-5 highcharts-tracker" style="" transform="translate(109,70)"></g>
        <g class="highcharts-label highcharts-data-label highcharts-data-label-color-6 highcharts-tracker" style="" transform="translate(170,40)"></g>
    </g>
    <g class="highcharts-label highcharts-tooltip highcharts-color-2" style="cursor:default;pointer-events:none;white-space:nowrap;" transform="translate(282,-9999)" opacity="0" visibility="visible">
        <path fill="rgba(255, 255, 255, 0)" class="highcharts-label-box highcharts-tooltip-box" d="M 0 0 L 125 0 C 125 0 125 0 125 0 L 125 102 C 125 102 125 102 125 102 L 0 102 C 0 102 0 102 0 102 L 0 0 C 0 0 0 0 0 0"></path>
    </g>
</svg>