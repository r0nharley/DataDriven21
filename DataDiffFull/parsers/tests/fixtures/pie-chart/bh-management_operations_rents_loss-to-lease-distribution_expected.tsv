#6eda55_0	Percentage	22.7909878508
#000000_0	Percentage	3.40591578217
#00cee6_0	Percentage	27.3746502118
#9b9bd7_0	Percentage	20.8174665564
#218A8C_0	Percentage	11.6023953514
#fbb755_0	Percentage	6.19112728627
#fc7570_0	Percentage	7.70309924565